fastchat/serve/openai_api_server.py

```diff
@@ -564,24 +594,45 @@ async def generate_completion(payload: Dict[str, Any]):
 
 
 @app.post("/v1/embeddings")
-async def create_embeddings(request: EmbeddingsRequest):
+@app.post("/v1/engines/{model_name}/embeddings")
+async def create_embeddings(request: EmbeddingsRequest, model_name: str = None):
     """Creates embeddings for the text"""
+    if request.model is None:
+        request.model = model_name
     error_check_ret = await check_model(request)
     if error_check_ret is not None:
         return error_check_ret
-    payload = {
-        "model": request.model,
-        "input": request.input,
-    }
 
-    embedding = await get_embedding(payload)
-    data = [{"object": "embedding", "embedding": embedding["embedding"], "index": 0}]
+    request.input = process_input(request.model, request.input)
+
+    data = []
+    token_num = 0
+    batch_size = 5
```

Comment:
```suggestion
    batch_size = 4
```

Reply:
Move this to `constant.py`

Reply:
> 

could be larger, not sure what's the appropriate number. I ran it with batch_size = 128 without any issue.